R/readNewClimate.R

```diff
@@ -59,6 +57,30 @@ readNewClimate <- function(subtype, subset) {
       toolProcessClimateTargetDatabase(database = "NewClimate", subtype = subtype)
 
     x <- as.magpie(input, spatial = "ISO_Code", temporal = "Target_Year")
+    # read in energy share targets from policy modeling protocol
+  } else if (grepl("RenShareTargets", subtype, fixed = TRUE)) {
+    data <- readxl::read_excel(
```

Comment:
did you update the source file without updating the file version? better increase the file version when you add content to the source to ensure backwards compatability